README.md

```diff
@@ -211,6 +211,7 @@ Here is a complete example:
 
 ```julia
 using Mux
+using HTTP.WebSockets: send
```

Comment:
```suggestion
using Mux.WebSockets: send
```

Or

```suggestion
using Mux.WebSockets
```

Might work and would match the client example.

I don't mind if the client example is changed to use HTTP.WebSockets or if this line is changed, but I think they should be consistent.